packages/auth/test/auth.spec.ts

```diff
@@ -439,7 +432,7 @@ describe('AuthTest', () => {
     describe('error handling', () => {
       // when allow_access is not implemented, the groups must match
       // exactly with the packages access group
-      test('should fails with bad password if adduser is not implemented', async () => {
+      test('should not fail if adduser is not implemented', async () => {
```

Comment:
@mbtools I need more context, why should not fails if is not implemented? 🤔

Reply:
If we fail, it will make implementing `adduser` mandatory. We can decide to keep failing, but it will stop old plugins from working (as described in #3734)